### PR TITLE
Spark `PlannerSpec` tests are now more robust

### PR DESCRIPTION
**Pull request by jedesah**
I did this to get the tests to pass on my Linux machine. Presumably the different jmv was causing the results to be returned in a different order. Now, the tests are agnostic to order.

### CODE REVIEW

File: sparkcore/src/test/scala/quasar/physical/sparkcore/fs/PlannerSpec.scala [hunk 1]
```diff
@@ -121,12 +121,12 @@ class PlannerSpec extends quasar.Qspec with QScriptHelpers with DisjunctionMatch
           state.eval(sc).run.map(result => result must beRightDisjunction.like{
             case rdd =>
               val results = rdd.collect
-              results.size must_= 5
-              results(0) must_= Data.Str("Poland")
-              results(1) must_= Data.Str("Poland")
-              results(2) must_= Data.Str("Poland")
-              results(3) must_= Data.Str("US")
-              results(4) must_= Data.Str("Austria")
+              results.toList must contain(exactly(
```
Comment (jedesah) [inline sparkcore/src/test/scala/quasar/physical/sparkcore/fs/PlannerSpec.scala]: Nice catch! I probably changed it mechanically when going through the file even though hopefully the test was not one of the ones that were failing. I certainly see why order makes sense for this one.

**Comment by jedesah** [comment]
java not found...

I'm just going to go ahead and restart that job and hope for the best 😟

**Comment by jedesah** [comment]
euh...

@drostron Any idea what's going on?

**Comment by drostron** [comment]
@jedesah Sounds like java went missing for the slamdata build as well.

**Comment by drostron** [comment]
@jedesah Travis is back to finding java again.

**Comment by jedesah** [comment]
I didn't even know the SlamData build uses Java! I'm glad it's fixed now.

**Pull request merged**